src/YieldDistributor.sol

```diff
@@ -110,17 +112,20 @@ contract YieldDistributor is IYieldDistributor, OwnableUpgradeable {
      * @return uint256 The voting power of the user
      */
     function getCurrentVotingPower(address _account) public view returns (uint256) {
-        return this.getVotingPowerForPeriod(BREAD, previousCycleStartingBlock, lastClaimedBlockNumber, _account)
-            + this.getVotingPowerForPeriod(BUTTERED_BREAD, previousCycleStartingBlock, lastClaimedBlockNumber, _account);
+        uint256 lastCycleStart = lastClaimedBlockNumber - cycleLength;
```

Comment:
Just realizing this will be invalid if `cycleLength` is changed. We need a "lastCycleLength" variable. Maybe it'd be good to create a `Cycle` struct?

Reply:
We should open a separate issue for this ... also mark it as bug so we don't upgrade changes without considering this...

Reply:
This is resolved by `previousCycleStartingBlock` , we no longer calculate the `lastCycleStart` variable, introduced this change from upstream (was introduced with #102)